src/analyses/variable-sensitivity/write_stack_entry.cpp

```diff
@@ -71,7 +71,7 @@ exprt offset_entryt::get_access_expr() const
   // This constructs a something that is basicallyt '(null)[offset])'
   // meaning that we don't know what the type is at this point, as the
   // array part will be filled in later.
-  return index_exprt(nil_exprt(), offset->to_constant());
+  return index_exprt::unchecked(nil_exprt(), offset->to_constant());
```

Comment:
While intentional as per the comment, is there perhaps a way of rewriting this code?

Reply:
Sure there is; I tried and gave up.

Reply:
I have added a commit doing this rewrite.